contents/stats.md

```diff
@@ -0,0 +1,445 @@
+# Statistics {#sec:stats}
+
+This content was created in order to awaken the reader to the importance of statistics for science and knowledge generation.
+Our idea is to present the concepts in the way we would like to have been provided when students about to be initiated into science.
+Our approach is to simplify the concepts as much as possible without losing their essence.
+**In this chapter, we will cover**:
+
+* what is **statistics**
+* the difference between **descriptive statistics** and **inferential statistics**
+* measures of **central tendencies**
+* measures of **dispersion**
+* measures of **dependence**
+* **probability distributions**
+* statistical **visualization**
+
+Statistics is important because it is a **tool to make sense of data**.
+With the abundant availability of data, we are often overwhelmed by numbers.
+Statistics offers a way to comprehend, summarize and infer information from data.
+We believe that every data scientist should have a basic understanding of statistics and how to perform simple statistical operations.
+
+We can divide statistics into two broad categories: **descriptive** and **inferential**.
+**Descriptive statistics summarizes and quantifies the characteristics of a given observed data**.
+Common metrics are: mean, median, mode, standard deviation, variance, correlation, percentiles.
+
+**Inferential statistics allows generating inferences** (statements) **from observed data about the data generation process**.
+All phenomena have an underlying data generating process that describes how the data is being generated.
+For example, in a soccer game, a scored goal can be explained by an underlying process: a tactic, error, struck of luck; or a mix of those.
+If we know a phenomenon's data generating process, we can use probability to simulate possible scenarios given certain aspects.
+Most of the time, specially in applied sciences, we do not have full knowledge of the data generating process.
+Given the observed data, we can retrace our way back to the data generating process.
+This process is known as **statistical inference**.
+Given some data, we can *infer* what are the aspects of the underlying data generating process.
+This is the realm of inferential statistics.
+
+In @fig:stats_vs_prob we summarize the relationship between data generating process and observed data.
```

Comment:
As a reader with a fresh mind, I would find the image useful a bit earlier (somewhere in the previous paragraph)

Reply:
Let's hear @lazarusA's opinion also...

Reply:
Yes, a little bit earlier will be better. See suggestions.